website/content/v1.10/talos-guides/install/single-board-computers/rock5b.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "Radxa ROCK 5B"
+description: "Installing Talos on Radxa ROCK 5B SBC using raw disk image."
+aliases:
+  - ../../../single-board-computers/rock5b
+---
+
+## Prerequisites
+
+You will need
+
+- `talosctl`
+- an SD card
+
+Download the latest `talosctl`.
```

Comment:
Can you please link people to this install page instead of putting manual commands here. It'll help us maintain the instructions. https://www.talos.dev/v1.9/talos-guides/install/talosctl/